statusboard/src/components/Header/Header.scss

```diff
@@ -4,7 +4,7 @@
   display: flex;
   flex-direction: row;
   align-items: center;
-  justify-content: space-between;
+  justify-content: flex-start;
```

Comment:
**Fix hamburger:** We need `space-between` because the way they hide the drop-down is by putting it to the right of that button. If it isn't in the right end of the header part of it will be visible.

Reply:
I hope it is possible to put in the section for small screens?